pkg/cmd/kind/create/cluster/createcluster.go

```diff
@@ -83,7 +83,7 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 		&flags.Wait,
 		"wait",
 		time.Duration(0),
-		"wait for control plane node to be ready (default 0s)",
+		"the length of time to wait for control plane node to be ready (default 0s). Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests.",
```

Comment:
```suggestion
		"the length of time to wait for control plane node to be ready (default 0s). Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't wait.",
```
(I see this is 1:1 from kubectl, we are doing something slightly different here so one small tweak)